opta/commands/deploy.py

```diff
@@ -129,22 +130,25 @@ def deploy(
         outputs = Terraform.get_outputs(layer)
     except MissingState:
         outputs = {}
-    if "docker_repo_url" not in outputs or outputs["docker_repo_url"] == "":
-        logger.info(
-            "Did not find docker repository in state, so applying once to create it before deployment"
-        )
-        _apply(
-            config=config,
-            env=env,
-            refresh=False,
-            image_tag=None,
-            test=False,
-            local=local,
-            auto_approve=auto_approve,
-            stdout_logs=False,
-            detailed_plan=detailed_plan,
-        )
-    image_digest, image_tag = _push(image=image, config=config, env=env, tag=tag)
+
+    image_digest, image_tag = (None, None)
+    if custom_image_name == "AUTO":
```

Comment:
can be lower case

Reply:
More so...let me make this case insensitive.